src/components/NFTCollection/CollectionItem.css

```diff
@@ -0,0 +1,58 @@
+.collection-item {
+    display: flex;
+    height: 50%;
+    flex-direction: row;
+    flex: 3;
+    justify-content: space-between;
+    align-items: left;
+    padding: 0.5rem;
+    margin: 1rem 0;
+    background-color: #ba5a5a;
+  }
+  
+  .collection-item__description {
+    display: flex;
+    height: 60%;
+    flex-direction: row;
+    gap: 1rem;
+    align-items: flex-end;
+    flex-flow: column-reverse;
+    justify-content: flex-start;
+    flex: 5;
+  }
+  
+  .collection-item h2 {
+    color: #3a3a3a;
+    font-size: 1rem;
+    flex: 1;
+    margin: 0 1rem;
+    color: white;
+  }
+  
+  .collection-item__price {
+    font-size: 1rem;
+    font-weight: bold;
+    color: white;
+    background-color: #40005d;
+    border: 1px solid white;
+    padding: 0.5rem;
+    border-radius: 12px;
+  }
+  
+  @media (min-width: 100) {
```

Comment:
Why did you decide to use 100px? Or is it a temporary placeholder until we work a bit more on responsiveness later on?

Reply:
Yep, planned to use it as a temporary placeholder. Edited to get rid of the number.